internal/container/default_registry.go

```diff
@@ -99,13 +111,19 @@ func (r Registry) HostFromCluster() string {
 // replaceRegistry produces a new image name that is in the specified registry.
 // The name might be ugly, favoring uniqueness and simplicity and assuming that the prettiness of ephemeral dev image
 // names is not that important.
-func replaceRegistry(defaultReg string, rs RefSelector) (reference.Named, error) {
+func (r Registry) replaceRegistry(defaultReg string, rs RefSelector) (reference.Named, error) {
 	if defaultReg == "" {
 		return rs.AsNamedOnly(), nil
 	}
 
 	// validate the ref produced
-	newNs := fmt.Sprintf("%s/%s", defaultReg, escapeName(rs.RefFamiliarName()))
+	newNs := ""
```

Comment:
this wasn't your variable name but while you're here--this looks like `newNamespace` and took me a sec to process, maybe rename?

Reply:
i also wondered about this! fixed